### ISSUE TITLE
Bad cfgs for non-x86 architectures with `linux_latest`

### ISSUE THREAD

**Issue by SUPERCILEX** [bytecodealliance/rustix#958]
```
error[E0425]: cannot find function `zero` in this scope
   --> /cargo/registry/src/index.crates.io-6f17d22bba15001f/rustix-0.38.26/src/backend/linux_raw/fs/syscalls.rs:182:13
    |
182 |             zero()
    |             ^^^^ not found in this scope
    |
help: consider importing this function
    |
9   + use crate::backend::conv::zero;
    |
```

See https://github.com/SUPERCILEX/forkfs/actions/runs/7079041249/job/19265244765

**Comment by SUPERCILEX** [comment]
Looks like this commit introduced the bug: https://github.com/bytecodealliance/rustix/commit/eb774c1da0304c72b9af6a3d0b2e802890ca7af8#diff-41472e367c17660a4d41317e747a725ab5b88fab02307fca7869a2d7498329cf

Needs this cfg: https://github.com/bytecodealliance/rustix/blob/2a70730e7de3c58a99b210432a164be429216322/src/backend/linux_raw/fs/syscalls.rs#L173

---

The real issue is that this wasn't caught by CI. Since we don't want combinatorial explosions, I'd say maybe just add `linux_latest` feature CI tests for x86_64, aarch64, and rvgc64.

**Comment by sunfishcode** [comment]
This is now fixed in rustix 0.38.27.
